### ISSUE TITLE
Sound Crack in Rayman PS1

### ISSUE THREAD

**Issue by Thexterax** [stenzek/duckstation#1829]
So, in Rayman PS1, sometimes there's this audio crack. A few people have noticed it so far which was not present in any other emulator.

https://clips.twitch.tv/WrongCuteLardTBTacoRight-6Nc9oaIM-TJbEv8x

Clip about it happening here if that's of any use.

Sometimes it just randomly starts happening. usually when the emulator has been on for a while.

Can fix the problem by using a save state > load state.

First we tried the "cubeb" sound device and then switched to SDL. But this seems irrelevant to the problem.

That said, many people have not reported any issues.

**Comment by andercard0** [comment]
> So, in Rayman PS1, sometimes there's this audio crack. A few people have noticed it so far which was not present in any other emulator.
> 
> https://clips.twitch.tv/WrongCuteLardTBTacoRight-6Nc9oaIM-TJbEv8x
> 
> Clip about it happening here if that's of any use.
> 
> Sometimes it just randomly starts happening. usually when the emulator has been on for a while.
> 
> Can fix the problem by using a save state > load state.
> 
> First we tried the "cubeb" sound device and then switched to SDL. But this seems irrelevant to the problem.
> 
> That said, many people have not reported any issues.

Please, FILL THE TEMPLATE

Please read before creating a new issue:

1. Make sure there is not already an open issue for this bug.
2. All enhancements **must** be off. To quickly disable all enhancements with affecting your normal config, in advanced options you can check "Disable All Enhancements".
3. All advanced options **must** be at their default values.
4. No cheats may be active. If you were using cheats, they should be disabled, and the game rebooted before reporting the bug.
5. Do not share save state files. Memory cards are okay.
6. Verify your dump, as we can not assist with issues resulting from bad dumps.
7. If playing PAL region software, please check whether or not the game has LibCrypt protection and that if it does, you have a correct SBI file (see https://github.com/stenzek/duckstation#libcrypt-protection-and-sbi-files for more information).
8. Please post your issue report in English as unfortunately this is the only language spoken by the developers. The Discord server has many helpful people if you need help translating.

**Remove everything before and including this line before submitting.**

**Game details**
[Serial Code, Region]
[i.e SLUS-00404 Ace Combat 2 (USA)]

**Description of the issue / bug**
[Describe what you are seeing and/or hearing during gameplay]

**Controller Troubleshoot Report**
1. Have you installed any drivers or wrappers on your system, or do you have any programs like Steam open?
2. Which controller backend are you using in general settings

**Note:**
If you are using Duckstation on Android device please consider the following before report:
Are you using any kind of "memory optimizer" program such as: Ccleaner, Wisecleaner, Clean Master, Boost Android etc. ?
(if so, please, consider create an exception on it first and test; or even remove / uninstall it retest and then, if the problem persists continue with the report.

**Steps to reproduce**
[Try to provide as much detail as possible to reproduce the issue]

**Enhancements information**
[Make sure they are all turned off before report]

**Hardware/software**
[If Android, which phone and Android version]
[If desktop, your CPU, graphics card, and operating system]
[GPU Renderer - D3D11/OpenGL/Vulkan]

**Emulator version**
[Shown in the title bar of the emulator]

**Additional context**
[Add any other context about the problem here]

**Comment by Thexterax** [comment]
My bad. Hopefully this is  better:
> 
> **Game details**
> [Serial Code, Region]
[SLUS-00005 Rayman (USA)]
> 
> **Description of the issue / bug**
So, in Rayman PS1, sometimes there's this audio crack. A few people have noticed it so far which was not present in any other emulator.
https://clips.twitch.tv/WrongCuteLardTBTacoRight-6Nc9oaIM-TJbEv8x
Clip about it happening here if that's of any use.
Sometimes it just randomly starts happening. usually when the emulator has been on for a while.
Can fix the problem by using a save state > load state.
First we tried the "cubeb" sound device and then switched to SDL. But this seems irrelevant to the problem.
That said, many people have not reported any issues.
> 
> **Controller Troubleshoot Report**
> 
> 1. Have you installed any drivers or wrappers on your system, or do you have any programs like Steam open?
> 2. Which controller backend are you using in general settings
No

> **Note:**
> If you are using Duckstation on Android device please consider the following before report:
> Are you using any kind of "memory optimizer" program such as: Ccleaner, Wisecleaner, Clean Master, Boost Android etc. No
> (if so, please, consider create an exception on it first and test; or even remove / uninstall it retest and then, if the problem persists continue with the report.
> 
> **Steps to reproduce**
> [Try to provide as much detail as possible to reproduce the issue]
It kinda randomly happens as said before... Tried pinpointing it before.
> **Enhancements information**
> [Make sure they are all turned off before report]
> 
> **Hardware/software**
> [If Android, which phone and Android version]
> [If desktop, your CPU, graphics card, and operating system]
> [GPU Renderer - D3D11/OpenGL/Vulkan]
D3D11 but I'm not sure if a sound issue has anything to do with it. Windows 10
> **Emulator version**
> [Shown in the title bar of the emulator]
gb1cf2555
> **Additional context**
> [Add any other context about the problem here]

**Comment by stenzek** [comment]
You still didn't include all the information in the template...

Will need a memory card and detailed instructions on how to reproduce before I can investigate.

**Comment by Thexterax** [comment]
> You still didn't include all the information in the template...
> 
> Will need a memory card and detailed instructions on how to reproduce before I can investigate.

All I can tell you is that it doesn't seem to happen right after launcing the game but only after emu's been on for a while.

Aside from this, it's random. I haven't experienced it myself yet. But 3 people in total so far have reported the problem. For 2 of them, this happened a while ago and they lost the twitch broadcast.

**Comment by stenzek** [comment]
I doubt streaming/third-party programs would cause any issues, but maybe if they're injecting their own DLLs in.

Regardless, until you can narrow down a cause, there's no much I can do. It's impossible to fix an issue like this without being able to reproduce and debug it myself.
